src/lib/holocene/select/select.svelte

```diff
@@ -96,7 +96,12 @@
         <Icon name="lock" class="text-gray-500" />
       {/if}
     </MenuButton>
-    <Menu id="{id}-menu" class="h-auto max-h-96 min-w-fit" {show} {dark}>
+    <Menu
+      id="{id}-menu"
+      class="h-auto max-h-96 overflow-y-scroll min-w-fit"
+      {show}
+      {dark}
```

Comment:
Perhaps an issue for another PR, but it looks like the `dark` option isn't quite right ⬇️ 

![Screenshot 2023-04-19 at 10 15 42 AM](https://user-images.githubusercontent.com/15069288/233137118-811c0a68-5d0a-4b96-a86c-00b1acbc4f95.png)

Reply:
**Update:** I just wasn't setting `dark` on the `MenuItem` as well 🤦‍♀️ please disregard.